docs/guides/rate-limits.mdx

```diff
@@ -0,0 +1,24 @@
+---
+id: rate-limits
+title: Project Rate Limits
+sidebar_label: Rate Limits
+---
+
+Ory Network currently enforces three rate limit policies based on the path of your request.
+
+1. `/sessions/whoami`: a dedicated rate limit policy for session checks.
+2. `/identities`: a dedicated rate limit policy for identity inserts and updates.
```

Comment:
We have GET and POST on this path, the details are a bit more complex.
IMO fine for a first version though.